spec/components/review_draft/draft/view_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe ReviewDraft::Draft::View do
+  alias_method :component, :page
+
+  before do
+    render_inline(described_class.new(trainee: trainee))
+  end
+
+  context "sections that appear for all trainees regardless of route" do
+    let(:trainee) { create(:trainee) }
+
+    it "shows for all trainees" do
```

Comment:
Please could we also test that some sections _don't_ appear, so we're guarding against the bug that graham was speaking about this morning i.e. schools section, placement section

Reply:
i've added extra tests as you said to guard against the wrong sections rendering for wrong routes